gluonnlp/data/word_embedding_training.py

```diff
@@ -31,9 +31,6 @@
 from .utils import _get_home_dir
 
 
-###############################################################################
-# Datasets
-###############################################################################
 class Text8(CorpusDataset):
```

Comment:
Could text8 also be used for language model? Is this file the best place to put the dataset?

Reply:
The distinction between language model and "normal" corpora was removed in https://github.com/dmlc/gluon-nlp/pull/128 but had to be reverted due to missing consensus. We should revisit, but I didn't get back to doing it again.

Reply:
ok